tests/utils.py

```diff
@@ -178,6 +178,35 @@ def __new__(cls, name, description=None):
         return make_run_option_decorator(RunOption(name, description=description))
 
 
+class TestVersion(LooseVersion):
+    """
+    Special version supporting ``latest`` keyword to ignore safeguard check of :func:`warn_version` during development.
+
+    .. seealso::
+        Environment variable ``MAGPIE_TEST_VERSION`` should be set with the desired version or ``latest`` to evaluate
+        even new features above the last tagged version.
```

Comment:
Maybe the MAGPIE_TEST_VERSION could also be documented elsewhere (with an explanation similar to this comment), or added to the docker-compose.yml.example and/or magpie.env.example?

Reply:
Yes. I was waiting for you 'tests' section update to add it.